src/util/git-ref-resolver.js

```diff
@@ -38,39 +39,36 @@ class GitRefResolver {
 
   async resolve(): Promise<DefaultBranch | ResolvedSha | false> {
     const testFunctions = [
```

Comment:
Still defining a new array for each invocation and the array is actually the same for every invocation.